src/Cynthia.Card/src/Cynthia.Card.Common/CardEffects/DIY/Monsters/Copper/Hatching.cs

```diff
@@ -0,0 +1,36 @@
+using System.Linq;
```

Comment:
By the way, I would really love if in the future iterations, we could implement the changes by archetypes rather than taking a few cards from many archetypes. I mean to try to push 1 or 2 complete archetypes, as all cards here were designed to work as a package for each archetype. For instance the new "The Thing in The Swamp" serves little purpose without the additional fog sources the patch planned to introduce, or the self-wound buffs won't do much if we don't allow the units that are cursed AND damaged to count twice for the berserker marauder